src/scripts/main.js

```diff
@@ -1 +1,71 @@
 'use strict';
+
+const body = document.querySelector('body');
+const logoMA = document.querySelector('h1');
+
+const createNotification = function(message, className) {
+  const DIV = `<div data-qa="notification" class='${className}'>${message}</div>`;
```

Comment:
Try to fix all linter errors. If you have any questions - feel free to ask them in `fe_chat`